charts/tfy-clickhouse/values.yaml

```diff
@@ -58,6 +66,9 @@ clickhouse:
   imagePullSecrets: []
   podSecurityContext: {}
   containerSecurityContext: {}
+  serviceAccount:
+    create: true
+    name: tfy-clickhouse
```

Comment:
Please add a comment as to why we can't add the support for automount service token and paste the link of the upstream file.